addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -29,7 +29,7 @@
 import odoo.modules
 
 from odoo import _, api, models, fields
-from odoo.exceptions import UserError
+from odoo.exceptions import UserError, ValidationError
```

Comment:
No problem with this line ^^
Could you also remove 
* One of line 20 or 24 
* Line 26, if indeed not necessary anymore? It doesn't look like it but maybe something breaks?